opendevin/core/main.py

```diff
@@ -157,14 +157,13 @@ async def on_event(event: Event):
     else:
         raise ValueError('No task provided. Please specify a task through -t, -f.')
 
-    # Figure out the LLM config
+    # Override default LLM configs ([llm] section in config.toml)
     if args.llm_config:
         llm_config = get_llm_config_arg(args.llm_config)
         if llm_config is None:
             raise ValueError(f'Invalid toml file, cannot read {args.llm_config}')
-        llm = LLM(llm_config=llm_config)
-    else:
-        llm = LLM(llm_config=config.get_llm_config_from_agent(args.agent_cls))
+        config.set_llm_config(llm_config)
+    llm = LLM(llm_config=config.get_llm_config_from_agent(args.agent_cls))
```

Comment:
Does it mean that, if I still specify the model config for a particular agent (e.g., browsing), while also providing the default config - It will use the overrided default config (`--llm-config`) for other agents, and will use the agent-specific config in `config.toml` for browsing agent?

Reply:
Yep